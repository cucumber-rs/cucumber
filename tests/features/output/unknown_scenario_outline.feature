Feature: Outline

  Scenario Outline: foo
    Given foo is <bar1>
    When foo is <bar2>
    Then foo is <bar4>

    Examples:
      | bar1 | bar2 | bar3 |
      | 0    |  1   |  2   |
