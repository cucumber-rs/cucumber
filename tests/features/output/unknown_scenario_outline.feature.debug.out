ParsingError
ParsingFinished { features: 0, rules: 0, scenarios: 0, steps: 0, parser_errors: 1 }
Started
Finished
